### PR TITLE
Muki test2

### PR DESCRIPTION
**Pull request by Mukelangelo**
we test some stuff now

### CODE REVIEW

File: src/Menu.cpp [hunk 1]
```diff
@@ -18,9 +18,8 @@ Menu::Menu()
     //m_bg = sf::Sprite(*Resources::instance().getBackground(2));
 
     // set window icon 
-    auto icon = sf::Image{};
-    icon.loadFromFile("ogre.png");
-    m_window.setIcon(icon.getSize().x, icon.getSize().y, icon.getPixelsPtr());
+    //auto icon = sf::Image{};
```
Comment (michabri) [inline src/Menu.cpp:21]: why did you remove? please explain

**Review by michabri — APPROVED** [review]

File: src/main.cpp [hunk 2]
```diff
@@ -3,5 +3,5 @@
 int main()
 {
 	Menu().run();
-	return 0;
+	return 1;
```
Comment (michabri) [inline src/main.cpp:6]: change it to 0 please

**Review by michabri — CHANGES_REQUESTED** [review]

File: src/main.cpp (hunk 2 above)
Comment (Mukelangelo) [reply src/main.cpp:6]: ok done
